lib/components/user/terms-of-use-pane.js

```diff
@@ -1,44 +1,50 @@
-import React from 'react'
-import { connect } from 'react-redux'
 import { Checkbox, ControlLabel, FormGroup } from 'react-bootstrap'
+import { connect } from 'react-redux'
 import { FormattedMessage, useIntl } from 'react-intl'
+import React from 'react'
 
-import { TERMS_OF_SERVICE_PATH, TERMS_OF_STORAGE_PATH } from '../../util/constants'
+import { LinkOpensNewWindow } from '../util/externalLink'
+import {
+  TERMS_OF_SERVICE_PATH,
+  TERMS_OF_STORAGE_PATH
+} from '../../util/constants'
 
 /**
  * User terms of use pane.
  */
+/* eslint-disable react/prop-types */
+/* TODO: Prop Types */
```

Comment:
So the reason for the `eslint-disable` and the TODO is because the CI tests were failing! You can see them on [6267208](https://github.com/opentripplanner/otp-react-redux/pull/859/commits/6267208024300eb070dbcf60ed88a1a99e3bed7c). I wanted to impact this file as little as possible outside of the external links, but I suppose if we're already in there and we can change the types, maybe we should? Not sure what the best practice is so I'll defer to 2nd reviewer!!

Reply:
Yes we generally typescript non typescript files whenever we make major changes to them